telnet/src/BulletinsHandler.php

```
@@ -0,0 +1,286 @@
+                $visible = 0;
+            }
+
+            $current .= $line[$i];
+            $visible++;
+        }
+
+        $segments[] = $current . ($activeSgr !== '' ? "\033[0m" : '');
+        return $segments;
+    }
+
+    private function nextActiveSgr(string $activeSgr, string $sgr): string
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$activeSgr'. (undefined)

(UnusedFormalParameter)